src/datasets/features/image.py

```diff
@@ -41,6 +41,16 @@ class Image:
     Args:
         decode (:obj:`bool`, default ``True``): Whether to decode the image data. If `False`,
             returns the underlying dictionary in the format {"path": image_path, "bytes": image_bytes}.
+
+    Examples:
+
+    ```py
+    >>> from datasets import load_dataset, Image
+    >>> ds = load_dataset("beans", split="train")
+    >>> ds = ds.cast_column('image', Image(decode=False))
```

Comment:
I think we can also show the PIL Image object
```suggestion
    >>> ds.features["image"]
    Image(decode=True, id=None)
    >>> ds[0]["image"]
    <PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=500x500 at 0x15E52E7F0>
    >>> ds = ds.cast_column('image', Image(decode=False))
    >>> ds[0]["image"]
```